drivers/base/bootvid/i386/pc/pc.h

```diff
@@ -21,7 +21,7 @@ extern UCHAR PixelMask[8];
 
 FORCEINLINE
 VOID
-SetPixel(
+VidpSetPixel(
```

Comment:
```suggestion
SetPixel(
```
to emphasise it's a per-arch dependent global helper function and not a generic private bootvid helper.